plugins/GSdx/Renderers/DX11/GSDevice11.h

```diff
@@ -206,9 +217,10 @@ class GSDevice11 final : public GSDevice
 				uint32 read_ba:1;
 				uint32 fbmask:1;
 
-				// *** Word 2
 				// Blend and Colclip
+				uint32 accu_blend:2;
 				uint32 clr1:1;
+				uint32 hdr:1;
```

Comment:
If we aren't using the hdr part for the shader then we might as well revert/purge it.

Reply:
The HDR part might still be still be necessary for deciding when to clamp the color in  SW blending since I recently ported over a little bit more of the SW blend from OGL. I actually removed the PS_ACCU_BLEND flag instead in the new commit to be closr to the OGL side. I will leave PS_HDR for now it in for now and see after I have cleaned things up if it can be removed.

Reply:
Ah yes with the recent changes it is useful now.